core/variant/variant.h

```diff
@@ -84,58 +85,75 @@ typedef Vector<Vector4> PackedVector4Array;
+	static constexpr Type OBJECT = VariantType::OBJECT;
+	static constexpr Type CALLABLE = VariantType::CALLABLE;
+	static constexpr Type SIGNAL = VariantType::SIGNAL;
+	static constexpr Type DICTIONARY = VariantType::DICTIONARY;
+	static constexpr Type ARRAY = VariantType::ARRAY;
+	static constexpr Type PACKED_BYTE_ARRAY = VariantType::PACKED_BYTE_ARRAY;
+	static constexpr Type PACKED_INT32_ARRAY = VariantType::PACKED_INT32_ARRAY;
+	static constexpr Type PACKED_INT64_ARRAY = VariantType::PACKED_INT64_ARRAY;
+	static constexpr Type PACKED_FLOAT32_ARRAY = VariantType::PACKED_FLOAT32_ARRAY;
+	static constexpr Type PACKED_FLOAT64_ARRAY = VariantType::PACKED_FLOAT64_ARRAY;
+	static constexpr Type PACKED_STRING_ARRAY = VariantType::PACKED_STRING_ARRAY;
+	static constexpr Type PACKED_VECTOR2_ARRAY = VariantType::PACKED_VECTOR2_ARRAY;
+	static constexpr Type PACKED_VECTOR3_ARRAY = VariantType::PACKED_VECTOR3_ARRAY;
+	static constexpr Type PACKED_COLOR_ARRAY = VariantType::PACKED_COLOR_ARRAY;
+	static constexpr Type PACKED_VECTOR4_ARRAY = VariantType::PACKED_VECTOR4_ARRAY;
+	static constexpr Type VARIANT_MAX = VariantType::VARIANT_MAX;
+
+	using Operator = VariantOperator;
+	static constexpr Operator OP_EQUAL = VariantOperator::OP_EQUAL;
+	static constexpr Operator OP_NOT_EQUAL = VariantOperator::OP_NOT_EQUAL;
+	static constexpr Operator OP_LESS = VariantOperator::OP_LESS;
+	static constexpr Operator OP_LESS_EQUAL = VariantOperator::OP_LESS_EQUAL;
+	static constexpr Operator OP_GREATER = VariantOperator::OP_GREATER;
+	static constexpr Operator OP_GREATER_EQUAL = VariantOperator::OP_GREATER_EQUAL;
+	static constexpr Operator OP_ADD = VariantOperator::OP_ADD;
+	static constexpr Operator OP_SUBTRACT = VariantOperator::OP_SUBTRACT;
+	static constexpr Operator OP_MULTIPLY = VariantOperator::OP_MULTIPLY;
+	static constexpr Operator OP_DIVIDE = VariantOperator::OP_DIVIDE;
+	static constexpr Operator OP_NEGATE = VariantOperator::OP_NEGATE;
+	static constexpr Operator OP_POSITIVE = VariantOperator::OP_POSITIVE;
+	static constexpr Operator OP_MODULE = VariantOperator::OP_MODULE;
+	static constexpr Operator OP_POWER = VariantOperator::OP_POWER;
+	static constexpr Operator OP_SHIFT_LEFT = VariantOperator::OP_SHIFT_LEFT;
+	static constexpr Operator OP_SHIFT_RIGHT = VariantOperator::OP_SHIFT_RIGHT;
+	static constexpr Operator OP_BIT_AND = VariantOperator::OP_BIT_AND;
+	static constexpr Operator OP_BIT_OR = VariantOperator::OP_BIT_OR;
+	static constexpr Operator OP_BIT_XOR = VariantOperator::OP_BIT_XOR;
+	static constexpr Operator OP_BIT_NEGATE = VariantOperator::OP_BIT_NEGATE;
+	static constexpr Operator OP_AND = VariantOperator::OP_AND;
+	static constexpr Operator OP_OR = VariantOperator::OP_OR;
+	static constexpr Operator OP_XOR = VariantOperator::OP_XOR;
+	static constexpr Operator OP_NOT = VariantOperator::OP_NOT;
+	static constexpr Operator OP_IN = VariantOperator::OP_IN;
+	static constexpr Operator OP_MAX = VariantOperator::OP_MAX;
```

Comment:
Note that integrating C++20 (see #100749) would cut this verbose section down to just four lines:
```suggestion
	using enum VariantType;
	using Type = VariantType;
	using enum VariantOperator;
	using Operator = VariantOperator;
```